mappings/net/minecraft/client/realms/task/ResettingWorldTask.mapping

```diff
@@ -6,3 +6,6 @@ CLASS net/minecraft/class_4440 net/minecraft/client/realms/task/ResettingWorldTa
 		ARG 1 serverId
 		ARG 3 title
 		ARG 4 callback
+	METHOD method_32517 resetWorldWithSeed (Lnet/minecraft/class_4341;J)V
+		ARG 1 realms
+		ARG 2 seed
```

Comment:
```suggestion
	METHOD method_32517 resetWorld (Lnet/minecraft/class_4341;J)V
		ARG 1 client
		ARG 2 worldId
```
One of the implementations of this method `ResettingWorldTemplateTask.method_32517` doesn't seem to use seed.